client/src/components/Trainings.vue

```diff
@@ -1,18 +1,62 @@
 <template>
-  <styled-title>
-    Trainings
-    <styled-button>Add new training</styled-button>
-  </styled-title>
+  <styled-wrapper>
+    <styled-title>
+      Trainings
+      <styled-button>Add new training</styled-button>
+      <styled-sort-link href="/">Sort by: ???</styled-sort-link>
+    </styled-title>
+    <styled-list>
+      <styled-list-item>
+        <styled-list-title>Contest #64747</styled-list-title>
+        <div>23 sessions</div>
+      </styled-list-item>
+    </styled-list>
+  </styled-wrapper>
 </template>
 
 <script>
-import { Button, Title } from '../common/styledComponents';
+import styled from 'vue-styled-components';
+import {
+  Button,
+  TitleH1,
+  Section,
+  List,
+  ListItem,
+  ListItemTittle,
+} from '../common/styledComponents';
+
+const Title = styled(TitleH1)`
+  margin-bottom: 25px;
+  display: flex;
+  align-items: center;
+`;
+
+const AddTraining = styled(Button)`
+  margin-left: 45px;
+`;
+
+const SortLink = styled.a`
+  margin-left: auto;
+  color: #222222;
+  font-size: 18px;
+  text-decoration: none;
+  font-weight: 500;
+
+  &:focus {
```

Comment:
It is super cool to have focus state! Awesome! 👍

Reply:
Yeah it was nice try)) my fault thanks)